installing/installing_openstack/installing-openstack-user-sr-iov.adoc

```diff
@@ -0,0 +1,92 @@
+include::modules/installation-osp-accessing-api-floating.adoc[leveloffset=+2]
+include::modules/installation-osp-accessing-api-no-floating.adoc[leveloffset=+2]
+include::modules/installation-osp-describing-cloud-parameters.adoc[leveloffset=+1]
+include::modules/installation-initializing.adoc[leveloffset=+1]
+include::modules/installation-configuration-parameters.adoc[leveloffset=+1]
+include::modules/installation-osp-config-yaml.adoc[leveloffset=+2]
+include::modules/installation-osp-custom-subnet.adoc[leveloffset=+2]
+include::modules/installation-osp-fixing-subnet.adoc[leveloffset=+2]
+include::modules/installation-osp-emptying-worker-pools.adoc[leveloffset=+2]
+include::modules/installation-user-infra-generate-k8s-manifest-ignition.adoc[leveloffset=+1]
+include::modules/installation-osp-converting-ignition-resources.adoc[leveloffset=+1]
+include::modules/installation-osp-creating-control-plane-ignition.adoc[leveloffset=+1]
+include::modules/installation-osp-creating-network-resources.adoc[leveloffset=+1]
+Optionally, you can use the the `inventory.yaml` file that you created to customize your installation. For example, you can deploy a cluster that uses bare metal machines.
+
+include::modules/installation-osp-deploying-bare-metal-machines.adoc[leveloffset=+2]
+include::modules/installation-osp-creating-bootstrap-machine.adoc[leveloffset=+1]
+include::modules/installation-osp-creating-control-plane.adoc[leveloffset=+1]
+include::modules/cli-logging-in-kubeadmin.adoc[leveloffset=+1]
+include::modules/installation-osp-deleting-bootstrap-resources.adoc[leveloffset=+1]
+include::modules/installation-osp-configuring-sr-iov.adoc[leveloffset=+1]
+include::modules/installation-osp-creating-sr-iov-compute-machines.adoc[leveloffset=+1]
+include::modules/installation-approve-csrs.adoc[leveloffset=+1]
+include::modules/installation-osp-verifying-installation.adoc[leveloffset=+1]
+The cluster is operational. Before you can configure it for SR-IOV networks, though, you must perform additional tasks.
+
+include::modules/networking-osp-preparing-for-sr-iov.adoc[leveloffset=+1]
+include::modules/networking-osp-enabling-metadata.adoc[leveloffset=+2]
+include::modules/networking-osp-enabling-vfio-noiommu.adoc[leveloffset=+2]
+
+[NOTE]
+====
+After you apply the machine config to the machine pool, you can xref:../../post_installation_configuration/machine-configuration-tasks.html#checking-mco-status_post-install-machine-configuration-tasks[watch the machine config pool status] to see when the machines are available.
+====
+
+// TODO: If bullet one of Next steps is truly required for this flow, these topics (in full or in part) could be added here rather than linked to. 
+// This document is quite long, however, and operator installation and configuration should arguably remain in their their own assemblies. 
+
+The cluster is installed and prepared for SR-IOV configuration. You must now perform the SR-IOV configuration tasks in "Next steps."
```

Comment:
Contrary to my unpopular opinion, our docs have adopted the UK variant of placing the period outside quotes, so `"Next steps".` in this case.

Reply:
I had no idea. 😬

Reply:
```suggestion
The cluster is installed and prepared for SR-IOV configuration. You must now perform the SR-IOV configuration tasks in "Next steps".
```